File: frontend/src/services/org.js
import { get } from './apiClient'

export function listTeams() {
  return get('/api/org/teams')
}

export function listAdvisors(teamId) {
  const params = teamId ? `?team_id=${teamId}` : ''
  return get(`/api/org/advisors${params}`)
}

File: backend/routers/org.py
"""API endpoints for org hierarchy (teams, advisors)."""

from fastapi import APIRouter, Query, Request

router = APIRouter(prefix="/api/org", tags=["org"])


@router.get("/teams")
async def get_teams(request: Request):
    service = request.app.state.org_service
    return await service.list_teams()


@router.get("/advisors")
async def get_advisors(
    request: Request,
    team_id: str = Query(None),
):
    service = request.app.state.org_service
    return await service.list_advisors(team_id=team_id)
